packages/console/src/components/Tooltip/index.module.scss

```diff
@@ -1,25 +1,52 @@
 @use '@/scss/underscore' as _;
 
-.container {
+.tooltip {
   position: absolute;
-  transform: translate(-50%, -100%);
   border-radius: 8px;
   background: #191c1d;
   color: #e0e3e3;
   box-shadow: var(--light-shadow-s1);
   padding: _.unit(2) _.unit(3);
   font: var(--font-body-medium);
 
-  &::after {
+  &.tooltipDefault {
+    background: #191c1d;
+    color: #e0e3e3;
+  }
+
+  &.tooltipSuccess {
+    background: #008a71;
+    color: #fff;
```

Comment:
All colors in css should use the token variable defined in `_colors.scss`, for instance, `var(--color-primary)`.
Do NOT use hex code directly

Reply:
updated